requirements.txt

```diff
@@ -1,3 +1,4 @@
 selenium == 3.0.0b2
 flake8
 pytest == 2.9.2
+pytest-selenium
```

Comment:
pytest and pytest-selenium are test requirements, and should not be installed as dependencies for FoxPuppet. I would recommend listing FoxPuppet dependencies in `setup.py` and using Tox to run your tests, listing test dependencies in `tox.ini`.